spec/features/admin/order_cycles/simple_spec.rb

```diff
@@ -12,53 +12,58 @@
 
   scenario "updating many order cycle opening/closing times at once", js: true do
     # Given three order cycles
-    oc1 = create(:simple_order_cycle)
-    oc2 = create(:simple_order_cycle)
+    oc1 = create(:simple_order_cycle,
+                  orders_open_at: Time.zone.local(2000, 12, 12, 12, 12, 0),
+                  orders_close_at: Time.zone.local(2041, 12, 12, 12, 12, 1))
+    oc2 = create(:simple_order_cycle,
+                  orders_open_at: Time.zone.local(2000, 12, 12, 12, 12, 2),
+                  orders_close_at: Time.zone.local(2041, 12, 12, 12, 12, 3))
     oc3 = create(:simple_order_cycle,
-                 orders_open_at: Time.zone.local(2040, 12, 12, 12, 12, 12),
-                 orders_close_at: Time.zone.local(2041, 12, 12, 12, 12, 12))
+                 orders_open_at: Time.zone.local(2040, 12, 12, 12, 12, 4),
+                 orders_close_at: Time.zone.local(2041, 12, 12, 12, 12, 5))
 
     # When I go to the order cycles page
     login_as_admin_and_visit admin_order_cycles_path
 
-    # And I fill in some new opening/closing times and save them
-    within("tr.order-cycle-#{oc1.id}") do
-      find("input#oc#{oc1.id}_name").set "Updated Order Cycle 1"
-      find("input#oc#{oc1.id}_orders_open_at").set "2040-12-01 12:00:00"
-      find("input#oc#{oc1.id}_orders_close_at").set "2040-12-01 12:00:01"
+    ## -- OC1
+    # Manually fill out name
+    find("input#oc#{oc1.id}_name").set ""
+    fill_in("oc#{oc1.id}_name", :with => "Updated Order Cycle 1")
+    
+    ## -- OC2
+    # Manually fill out name
```

Comment:
IMO this particular comment that we repeat doesn't add much value. I think Capybara's DSL already conveys that.

Reply:
Agree.